Source/Core/CorridorGeometry.js

```diff
@@ -679,8 +679,8 @@ define([
 
     function computeRectangle(positions, ellipsoid, width, cornerType) {
         var cleanPositions = arrayRemoveDuplicates(positions, Cartesian3.equalsEpsilon);
-        var length = cleanPositions.length - 1;
-        if (length === 0 || width === 0) {
+        var length = cleanPositions.length;
```

Comment:
How hard is it to write a test for the updated code in this function.  One option would be exposing `computeRectangle` as a private `CorridorGeometry._computeRectangle` so that it can be tested directly.

CC @hpinkos